sdk/template/Azure.Template/CHANGELOG.md

```diff
@@ -1,106 +1,23 @@
 # Release History
 
-## 1.0.3-beta.19 (2020-09-24)
-- Test Submit-PR
-
-## 1.0.3-beta.18 (2020-09-24)
-- Test Submit-PR
-
-## 1.0.3-beta.17 (2020-09-16)
-- Package Artifacts directory restructure, attempt 7
-
-## 1.0.3-beta.16 (2020-09-15)
-- Package Artifacts directory restructure, attempt 6
-
-## 1.0.3-beta.15 (2020-09-15)
-- Package Artifacts directory restructure, attempt 5
-
-## 1.0.3-beta.14 (2020-09-14)
-- Package Artifacts directory restructure, attempt 4
-
-## 1.0.3-beta.13 (2020-09-14)
-- Package Artifacts directory restructure, attempt 3
-
-## 1.0.3-beta.12 (2020-09-11)
-- Package Artifacts directory restructure, attempt 2
-
-## 1.0.3-beta.11 (2020-09-10)
-- Package Artifacts directory restructure
-
-## 1.0.3-beta.10 (2020-09-09)
-- Enable the master link replacement feature, attempt 3
-
-## 1.0.3-beta.9 (2020-09-09)
-- Enable the master link replacement feature, attempt 2
-
-## 1.0.3-beta.8 (2020-09-09)
-- Enable the master link replacement feature, attempt 1
-
-## 1.0.3-beta.7 (2020-09-04)
-- Test release tag replacement
-
-## 1.0.3-beta.6 (2020-09-03)
-- Test new alpha beta versioning
-
-## 1.0.3-beta.5 (2020-09-03)
-- Test new alpha beta versioning
-
-## 1.0.3-beta.4 (2020-09-01)
-- Test new alpha beta versioning
-
-## 1.0.3-beta.3 (2020-08-31)
-- Test new alpha beta versioning
-
-## 1.0.3-beta.2 (2020-08-30)
-- Test new alpha beta versioning
-
-## 1.0.3-beta.1 (2020-08-27)
-- Test new alpha beta versioning
-
-## 1.0.2-preview.17 (2020-07-24)
-- Test release pipeline
-
-## 1.0.2-preview.15 (2020-06-29)
-- Test release pipeline
-
-## 1.0.2-preview.14 (2020-06-02)
-- Test release pipeline
-
-## 1.0.2-preview.13 (2020-03-23)
-- Test release pipeline
-
-## 1.0.2-preview.12 (2020-03-23)
-- Test release pipeline
-
-## 1.0.2-preview.11 (2020-03-23)
-- Test release pipeline
-
-## 1.0.2-preview.10 (2020-03-23)
-- Test release pipeline
-
-## 1.0.2-preview.9 (2020-03-17)
-- Test release pipeline
-
-## 1.0.2-preview.8 (2020-03-10)
-- Testing out release pipeline
-
-## 1.0.2-preview.7 (2020-03-06)
-- Testing out release pipeline
-
-## 1.0.2-preview.6 (2020-02-24)
-### Added
-- Testing Changelog added section
-- Testing Links [Project Site](https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/template/Azure.Template/README.md)
-- Test `Highlighted Code`
-
-## 1.0.2-preview.5 (2020-02-24)
-### Added
-- Testing Changelog added section
-- Testing Links [Project Site](https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/template/Azure.Template/README.md)
-- Test `Highlighted Code`
-
-## 1.0.2-preview.4 (2019-12-17)
-- Version Bump
-
-## 1.0.2-preview.3 (2019-12-17)
-- Started changelog to capture release notes.
+## 1.0.0-beta.1 (Unreleased)
```

Comment:
Would it make sense to preserve the history of the template in the CHANGELOG?  I'm actually not sure which is better, to show the history or provide a template for it.